i18n/validate.py

```diff
@@ -36,26 +40,46 @@ def validate_po_files(configuration, locale_dir, root_dir=None, report_empty=Fal
             if ext.lower() == '.po' and os.path.basename(filename) not in merged_files:
 
                 # First validate the format of this file
-                msgfmt_check_po_file(locale_dir, filename)
+                found_problems = msgfmt_check_po_file(locale_dir, filename) or found_problems
```

Comment:
I think I like a more spelled-out style for this line (and others like it):
```
msgfmt_found_problems = msgfmt_check_po_file(locale_dir, filename)
if msgfmt_found_problems:
    found_problems = True
```
This matches other places, and doesn't hide the critical "or found_problems" at the end of the line.

Reply:
Changed.